pgbouncer-mixin/panels.libsonnet

```diff
@@ -152,5 +154,48 @@ local utils = commonlib.utils;
           |||
         )
         + g.panel.timeSeries.standardOptions.withUnit('s'),
+
+      alertsPanel:
+        alertList.new('PgBouncer alerts')
+        + alertList.options.UnifiedAlertListOptions.withAlertInstanceLabelFilter(this.grafana.variables.queriesGroupSelectorAdvanced),
+
+      topDatabaseActiveConnection:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top database by active connections',
+          targets=[t.topDatabaseActiveConnection],
+          description=|||
+            Top databases by current number of active client connections.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('conn'),
+      topDatabaseQueryPooled:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top databases by queries processed',
+          targets=[t.topDatabaseQueryProcessed],
+          description=|||
+            Top databases by rate of SQL queries pooled by PgBouncer.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ops'),
+      topDatabaseQueryDuration:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top databases by average query duration',
+          targets=[t.topDatabaseQueryDuration],
+          description=|||
+            Top databases by average duration of queries being processed by PgBouncer.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('s'),
+      topDatabaseNetworkTraffic:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top databases by network traffic',
+          targets=[t.topDatabaseNetworkTrafficReceived, t.topDatabaseNetworkTrafficSent],
+          description=|||
+            Top databases by volume in bytes of network traffic received by PgBouncer.
```

Comment:
"...by volume of network traffic received by PgBouncer"

I wouldn't say "bytes" as this unit might change in the graph dynamically